### PR TITLE
Adding border-radius variable for LESS

### PR DESCRIPTION
**Pull request by Azaret**
PR for #1429

### CODE REVIEW

File: less/datepicker3.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .datepicker {
-	border-radius: 4px;
+	.border-radius(@border-radius-base);
```
Comment (acrobat) [inline less/datepicker3.less]: Hmm i'm not exactly sure why you added the 'dot' here? Or why it is in the BS2 file, should it be a "normal" property without the dot?

Comment (Azaret) [inline less/datepicker3.less]: Well the mixin is used on the other file, I guessed it made sense to use it here as well.
Now I see that there is no such mixin on the build, gonna fix this.

Comment (acrobat) [inline less/datepicker3.less]: Oh yes i forgot about the mixins! Thanks for the correction!

**Comment by acrobat** [comment]
Thanks @Azaret!

**Pull request merged**